cavityDisplayGUI.py

```diff
@@ -1,18 +1,23 @@
 from dataclasses import dataclass
+from functools import partial
+from typing import List
 
 from PyQt5.QtGui import QColor
-from PyQt5.QtWidgets import QHBoxLayout
+from PyQt5.QtWidgets import QHBoxLayout, QWidget
+
 from epics import PV
-from functools import partial
 from pydm import Display
 from pydm.widgets import PyDMEmbeddedDisplay, PyDMRelatedDisplayButton, PyDMTemplateRepeater
-from typing import List
 
-from cavityWidget import CavityWidget
 from lcls_tools.devices.scLinac import LINAC_OBJECTS
 
+import sys
+sys.path.insert(0, './frontend')
```

Comment:
I'm torn here because I kind of want to delete this and put the frontend in our python path

Reply:
I am not attached to this method; I just needed it to be functional! As long as it works, I'm okay either way:)